static/src/javascripts/projects/common/modules/save-for-later.js

```diff
@@ -82,6 +84,14 @@ define([
         return { prop74: prefix + 'ContainerSave:' + contentId };
     };
 
+    SaveForLater.prototype.conditionalInit = function(){
+        if (!liveBlog.notificationsCondition()) {
+            var $saveForLaterContainer = $('.meta__save-for-later');
+            $saveForLaterContainer.show();
```

Comment:
Also, try and use `js-` classes for getting elements and use `fastdom.write` when making style changes.